server/src/main/clients/web-browser/src/pages/projects/Overview/Overview.tsx

```diff
@@ -1,10 +1,21 @@
 import './Overview.scss';
 import {DefaultPage} from '../../../libs/DefaultPage/DefaultPage';
+import {ProjectDashboard} from '../../../libs/projects/ProjectDashboard/ProjectDashboard';
```

Comment:
I don't quite understand the pattern of having a single tag in a pages... that includes a single component in libs.... Is there a reason that you don't just have ProjectDashboard inlined in this file? Are you planning to use it elsewhere as well?

Reply:
Might we want to use it somewhere else?
One possibility is that all page sections are abstracted into components and all pages are simple and should mainly use other components

Reply:
We don't want to make something that we _may_ or _may not_ use somewhere else. Especially the page itself. If you have other, specific places in mind where you want to reuse something, or you are already reusing it on a page, just have it in the page directly. We can always refactor later.